Project-Hotpot/Project-Hotpot/ViewControllers/HomeViewController.swift

```diff
@@ -8,23 +8,27 @@
 import UIKit
 
 class HomeViewController: ViewController {
+    enum swipe{
+        case left
+        case right
+    }
+    
     var api_instance = SpotifyManager.shared()
     @IBOutlet weak var card: UIView!
     @IBOutlet weak var songImage: UIImageView!
     @IBOutlet weak var playButton: UIButton!
     
+    let songManager = SongManager()
     let pauseButtonImage = UIImage(systemName: "pause.circle.fill")
     let playButtomImage = UIImage(systemName: "play.circle.fill")
     
     @IBOutlet weak var songTitleLabel: UILabel!
     override func viewDidLoad() {
-        self.resetSong()
-        self.resetCard()
+        self.didSwipe(direction: swipe.left, completion: self.createClosure())
```

Comment:
Is this programmatically invoking swipe on view loading? If yes seems suboptimal, there should be a better way to do this like put the code in a shared function and call it from both swipe handler and here

EDIT: the function name didSwipe threw me off a bit. Maybe better name should be refreshSong or something